base/linalg/symmetric.jl

```diff
@@ -37,8 +37,7 @@ julia> Slower = Symmetric(A, :L)
  0  9  0  1  0
  2  0  3  0  4
 
-julia> eigfact(Supper)
-Base.LinAlg.Eigen{Float64,Float64,Array{Float64,2},Array{Float64,1}}([-2.96684,-2.72015,0.440875,7.72015,14.526],[-0.302016 -2.22045e-16 … 1.11022e-16 0.248524; -6.67755e-16 0.596931 … -0.802293 1.93069e-17; … ; 8.88178e-16 -0.802293 … -0.596931 0.0; 0.772108 8.93933e-16 … 0.0 0.630015])
+julia> eigfact(Supper);
```

Comment:
if we're not going to show the output, or some interesting subset of it, may as well just remove this line from the example